app/src/main/java/org/colourhood/instamessage/data.kt

```diff
@@ -0,0 +1,34 @@
+package org.colourhood.instamessage
```

Comment:
Do we need this file? Could you please double check and delete it if we don't need it.